_editorial_policies/01_bylaws.md

```diff
@@ -0,0 +1,168 @@
+            1. Diversity in career background, geographical perspectives, life experiences, and any other factors that are seen as beneficial to represent the broader community, provide fairness in decisions, and include a range of viewpoints.
+	    1. All members on the Editorial Board shall be elected in a fair, impartial, and confidential process without regard to race, nationality, religion, gender or gender identity, sexual orientation, or familial status.
+         1. Managing editors will be elected based on:
+            1. all criteria outlined for associate editors, plus demonstrated excellence serving as an associate editor.
+	 1. The Recording Secretary will be nominated by the Managing editors from among the associate editors.
+   1. Appointments Terms
+      1. Appointment for all positions are for three years, which may be extended for an unlimited 
+	 number of terms by a vote of two/thirds of the remaining Editorial Board. If the terms of more 
+         than one third of the Editorial Board end at the same time, all may remain in their positions for 
+         long enough for additional editors be appointed, or one month, which ever comes first. 
+      2. At the time the bylaws are adopted, the three managing editorial terms will be for two, three, and four years,
+         determined by mutual agreement or random chance if no agreement is reached.
+   1. Voting
+      1. All votes described in the bylaws are tallied based on the fraction
+         of the Editorial Board responding within one week written notice
+         via e-mail or other voting mechanism approved by the managing
+         editors. In exceptional circumstances advance notice may be given
+         of a rush decision, where the board may be given one week (or more)
+         notice that their votes will be needed within a shorter window,
+         such as a 12 hour window.
+
+1. Conflicts of interest
+   1. A 'conflict of interest' is defined as any of:
+      1. being a close co-author on a paper together for up to 48 months after publication. "Close co-author" is defined as serving as corresponding author and/or first author together, or equivalent close collaboration.  This definition was chosen to avoid overly restrictive limitations for papers with large numbers of loosely-associated contributing authors, and will have some leeway for interpretation as outlined below.
+      1. a co-awardee on a grant for up to 48 months after the grant, or that currently have a cooperative pending grant.
+      1. if there exists a postdoctoral or graduate advisor relationship at any time in the past.
+      1. at the same institution, or if there is a potential change of institution in progress that would result in being at the same institution.
+      1. having a business relationship (such as serving as a consultant for or part owner of same company) within 48 months.
```

Comment:
How about adding "as well as any other relationship which might affect one's ability to provide a fair and objective evaluation of a work, or might reasonably be expected to do so by an independent outside observer. When in doubt, potential conflicts should be disclosed."